CHANGELOG/v1.8.0.md

```diff
@@ -0,0 +1,524 @@
+# CAPI
+
+## Highlights
+
+- Bumped to controller-runtime v0.18, k8s.io/* v0.30, controller-gen v0.15, Go 1.22
+- API: Add support for negative polarity conditions (#10550)
+- API: Allow control plane provider to set control plane endpoint (#10667)
+- CABPK: Add kubeadm v1beta4 types (including ExtraEnvs and ImagePullSerial) (#10709, #10846)
+- CABPK: Allow to generate JoinConfiguration discovery kubeconfig (#10799)
+- ClusterClass: Improvements for variables
+  - CEL support for variables (#9239)
+  - Metadata for top-level and nested variables (& deprecate previous metadata field) (#10778)
+  - Support control plane variable overrides (#10682)
+  - New builtin variables: control plane / MD / MP metadata (#10837), Cluster UID (#10822)
+- KCP: Delete out of date machines with unhealthy control plane component conditions during rollout (#10196)
+- KCP: Default ControlPlaneKubeletLocalMode kubeadm feature gate to true for Kubernetes >= 1.31.0
+- Logging: Improvements to rollout logs: Cluster topology controller (#10690), KCP controller & webhooks (#10628), MachineDeployment controller (#10688)
+- MachineDeployment: Add remediation strategy (#10712)
+- MachineSet/Machine: Improvements to Machine deletion
+  - Machine deletion skips waiting for volumes detached for unreachable Nodes (#10662)
+  - Machine drain ignores terminating Pods earlier for unreachable Nodes (#10706)
+  - MachineSet: Fix deletion priority to avoid deleting too many machines (#10087)
+  - MachineSet: Deprioritize unknown NodeHealthy conditions for deletion (#10763)
+  - MachineSet: Ensure unhealthy machines get deletion priority (#10755)
+- A usual, significantly improved e2e test framework & coverage (#10083, #10216, #10135, #10498, #10639, #10530)
+- New providers in clusterctl: IONOS Cloud (#10767), Linode (#10471), Tinkerbell (#10487)
+
+## Deprecation Warning
+
+- ClusterClass: Deprecate definitionFrom field (#10841)
+- ClusterClass: Deprecate IP Family builtin variable (#10554)
+- MachineDeployment: Deprecate revision management (#10855)
+- MachineHealthCheck: Deprecate MaxUnhealthy & UnhealthyRange fields (#10853)
+
+Umbrella issue: API changes: #10852
+
+<details>
+<summary>More details about the release</summary>
```

Comment:
I am unsure about the drop-down, maybe we should put breaking changes and Kubernetes version support out side of the drop-down for more visibility.